virttest/utils_stress.py

```diff
@@ -197,3 +200,29 @@ def host_stress_event(self, event):
             else:
                 raise NotImplementedError
             time.sleep(self.itr_sleep_time)
+
+
+def install_stressapptest(vm):
+    """
+    Install stressapptest cmd
+
+    :param vm: the vm to be installed with stressapptest
+    """
+    session = vm.wait_for_login(timeout=360)
+    name = ["git", "gcc", "gcc-c++", "make"]
+    if not utils_package.package_install(name, session, timeout=300):
+        raise virt_vm.VMError("Installation of packages %s failed" % name)
```

Comment:
I don't think it is necessary for us to import virt_vm module to make more dependencies only because we need to raise an VMError. Here it is ok to simply raise exceptions.TestError.

Reply:
@dzhengfy updated, thx